dataops/dependencies/__init__.py

```diff
@@ -14,7 +14,8 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from dataops.dependencies.cache import Cache
-from dataops.dependencies.cache import get_redis
 from dataops.dependencies.db import get_db_session
+from dataops.dependencies.redis import RedisSingleton
+from dataops.dependencies.redis import get_redis
 
-__all__ = ['Cache', 'get_redis', 'get_db_session']
+__all__ = ['Cache', 'get_redis', 'get_db_session', 'RedisSingleton']
```

Comment:
I would suggest not to expose in `__all__` classes that should be directly used outside of dependencies.
```suggestion
__all__ = ['Cache', 'get_redis', 'get_db_session']
```

Reply:
You are right. Updated the `__all__` to just expose dependency functions to public.